Makefile

```diff
@@ -383,11 +383,11 @@ endif
 
 package-docs: build-docs build-docs-capi
 	mkdir -p "package/docs"
-	mkdir -p "package/docs/c"
+	mkdir -p "package/docs/c/runtime-c-api"
```

Comment:
It looks like we're removing the non-deprecated C docs from our releases here. We probably don't want to do that

Reply:
I think we are good:

```
package/docs
├── c
│   └── runtime-c-api
└── crates
    ├── implementors
    ├── src
    ├── test_generator
    ├── wasmer
    ├── wasmer_c_api
    ├── wasmer_cache
    ├── wasmer_cli
    ├── wasmer_compiler
    ├── wasmer_compiler_cranelift
    ├── wasmer_compiler_llvm
    ├── wasmer_compiler_singlepass
    ├── wasmer_derive
    ├── wasmer_emscripten
    ├── wasmer_engine
    ├── wasmer_engine_dummy
    ├── wasmer_engine_jit
    ├── wasmer_engine_native
    ├── wasmer_engine_object_file
    ├── wasmer_integration_tests_cli
    ├── wasmer_middlewares
    ├── wasmer_object
    ├── wasmer_types
    ├── wasmer_vm
    ├── wasmer_wasi
    ├── wasmer_wasi_experimental_io_devices
    └── wasmer_wast

29 directories

```

Reply:
What I meant is that the line

```
 	cp -R lib/c-api/doc/html package/docs/c-api 
 ```
 was changed to
 ```
  	cp -R lib/c-api/doc/deprecated/html/ package/docs/c/runtime-c-api 
  ```
  
  It looks like the crate-level C api is included in the docs, but I thought we had some other stuff too

Reply:
I think we should probably also rename `runtime-c-api` to `deprecated-api` or something, or otherwise make it more clear what it is!

Reply:
I kept the `runtime-c-api` name to not break existing links.

About the change on the `cp` command: the source directory changed, the doxygen part moved to another place so I had to change the command.

Reply:
Huh, it looks like we don't package docs into our releases anymore, I think we used to

Reply:
I guess it's fine then; I looked through the Makefile for the variable `SOURCE_VERSION` which is used in the package-docs command and as far as I can tell it doesn't exist.  I'm skeptical that it'll do the right thing -- it seems that code may now be out of date with respect to the rest of the Makefile, does the command do what you expect locally?